### PR TITLE
improving withdrawal guide

### PR DESCRIPTION
**Pull request by james-prysm**
based on user and team feedback on tasks being confusing for many users.

### CODE REVIEW

File: website/docs/wallet/withdraw-validator.md [hunk 1]
```diff
@@ -25,45 +25,25 @@ The **Capella/Shanghai Ethereum** upgrade lets you withdraw your validator nodes
 
 In this how-to, you'll learn how to perform both types of withdrawals.
 
+## In a nutshell
 
+Withdrawals are initiated by updating the `withdrawal_credentials` field on the validator with an Ethereum address of your choosing using your mnemonic.
```
Comment (rauljordan) [inline website/docs/wallet/withdraw-validator.md:30]:
```suggestion
Withdrawals are initiated by updating the `withdrawal_credentials` field on the validator to an Ethereum address of your choosing using your mnemonic. 
```

File: website/docs/wallet/withdraw-validator.md [hunk 2]
```diff
@@ -25,45 +25,25 @@ The **Capella/Shanghai Ethereum** upgrade lets you withdraw your validator nodes
 
 In this how-to, you'll learn how to perform both types of withdrawals.
 
+## In a nutshell
 
+Withdrawals are initiated by updating the `withdrawal_credentials` field on the validator with an Ethereum address of your choosing using your mnemonic. 
 
+**note: updating the `withdrawal_credentials` is irreversible, so you must carefully verify the Ethereum address you choose**
```
Comment (rauljordan) [inline website/docs/wallet/withdraw-validator.md:32]:
```suggestion
**note: updating the `withdrawal_credentials` is irreversible, so you must carefully verify the Ethereum address you wish to receive your funds in**
```

File: website/docs/wallet/withdraw-validator.md [hunk 3]
```diff
@@ -25,45 +25,25 @@ The **Capella/Shanghai Ethereum** upgrade lets you withdraw your validator nodes
...
 
-## Before you begin
+We will be covering the following steps in this guide:
 
-Familiarity with the following resources and terms will help you confidently complete this guide:
-
-<!-- TODO: These terms can now be moved into Glossary CMS and embedded via quicklooks to further streamline the content experience - ping Mick if you'd like to help with this. -->
+- download [staking-deposit-cli](https://github.com/ethereum/staking-deposit-cli)
+- gather validator information for the [staking-deposit-cli](https://github.com/ethereum/staking-deposit-cli)
+- run the [staking-deposit-cli](https://github.com/ethereum/staking-deposit-cli) in an `offline` environment with your mnemonic to generate the `blstoexecutionchange` message(s)
```
Comment (rauljordan) [inline website/docs/wallet/withdraw-validator.md:38]: I wonder how much we should say specific terms in this outline such as blstoexecutionchange jargon. Is there a way in which we could say something like "signed withdrawal request"?

File: website/docs/wallet/withdraw-validator.md [hunk 4]
```diff
@@ -25,45 +25,25 @@ The **Capella/Shanghai Ethereum** upgrade lets you withdraw your validator nodes
...
+- verify the `blstoexecutionchange` message(s) that the corresponding validator will set to the chosen Ethereum address and migrate the `blstoexecutionchange` message(s) back to an `online` environment
```
Comment (rauljordan) [inline website/docs/wallet/withdraw-validator.md:39]: maybe "signed withdrawal request"?

**Review by saolyn — APPROVED** [review]
Looks good

**Review by daisydocs — APPROVED** [review]

**Pull request merged**